test/customer.test.ts

```diff
@@ -0,0 +1,80 @@
+import { Wallet, ApplicationConfiguration, StellarConfiguration } from "../src";
+import { DefaultSigner } from "../src/walletSdk/Auth/WalletSigner";
+import { SigningKeypair } from "../src/walletSdk/Horizon/Account";
+import axios from "axios";
+import { getRandomString } from "./utils";
+
+let wallet;
+let accountKp;
+
+describe("Customer", () => {
+  beforeAll(async () => {
+    wallet = Wallet.TestNet();
+    const stellar = wallet.stellar();
+    const account = stellar.account();
+    accountKp = account.createKeypair();
+    await axios.get(
+      "https://friendbot.stellar.org/?addr=" + accountKp.publicKey,
+    );
+  }, 10000);
+
+  test("Sep-12 methods work", async () => {
+    const anchor = wallet.anchor({ homeDomain: "testanchor.stellar.org" });
+
+    const auth = await anchor.sep10();
+    const authToken = await auth.authenticate({ accountKp });
+
+    const sep12 = await anchor.sep12(authToken);
+    const customerType = "sep31-receiver";
+    const customerEmail = `${getRandomString(6)}@gmail.com`;
+
+    // Update
```

Comment:
Should the comment say "Add" instead of "Updated"? Since we have an "Update" on line 58

Reply:
yes ty https://github.com/stellar/typescript-wallet-sdk/pull/71/commits/e3f8ab3658106304826194b4c61f49b3ee6f7dc6